en/computer_vision/collision_prevention.md

```diff
@@ -106,53 +106,29 @@ Depending on the vehicle, type of environment and pilot skill different amounts
 Setting the [CP_GUIDE_ANG](#CP_GUIDE_ANG) parameter to 0 will disable the guidance, resulting in the vehicle only moving exactly in the directions commanded.
 Increasing this parameter will let the vehicle choose optimal directions to avoid obstacles, making it easier to fly through tight gaps and to keep the minimum distance exactly while going around objects.
 
-If this parameter is too small the vehicle may feel 'stuck' when close to obstacles, because only movement away from obstacles at minimum distance are allowed.
-If the parameter is too large the vehicle may feel like it 'slides' away from obstacles in directions not commanded by the operator.
+If this parameter is too small the vehicle may feel "stuck" when close to obstacles, because only movement away from obstacles at minimum distance are allowed.
+If the parameter is too large the vehicle may feel like it slides away from obstacles in directions not commanded by the operator.
 From testing, 30 degrees is a good balance, although different vehicles may have different requirements.
 
 ::: info
 The guidance feature will never direct the vehicle in a direction without sensor data.
-If the vehicle feels 'stuck' with only a single distance sensor pointing forwards, this is probably because the guidance cannot safely adapt the direction due to lack of information.
+If the vehicle feels stuck with only a single distance sensor pointing forwards, this is probably because the guidance cannot safely adapt the direction due to lack of information.
```

Comment:
FYI we only need to indicate something like "stuck" the first time, to highlight that we don't "exactly" mean stuck :-). I know this predates you, just wanted to explain why I did it.